ui/src/home-module/home-page/home-page.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="home-container">
+    <app-carousel-banner [caraouselData]="bannerData"></app-carousel-banner>
+    <div class="categories-container" [style.flex-direction]="isOdd ? 'row-reverse' : 'row'" *ngFor="
+    let category of categoriesData;
+    let i = index;
+    let isOdd = odd;
+    let isEven = even
+  ">
+        <div class="category-image">
```

Comment:
Shouldn't category be a separate component?

Reply:
Okay. Created new component of category banner.